lib/index.js

```diff
@@ -72,8 +72,11 @@ module.exports = {
     if (this.project.targets) {
       let browsers = browserslist(this.project.targets.browsers);
 
-      if (browsers.find(browser => browser.startsWith('ie'))) {
-        emberSourceDistPath = path.join(emberSourceDistPath, 'ie');
+      let hasIE = browsers.find(browser => browser.startsWith('ie'));
+      let hasLegacyAndroid = browsers.find(browser => browser.match(/android.*[1-4]\.[1-4]\.[1-4]/))
```

Comment:
what about e.g. `android 4.0.1`?

Reply:
yeah, discussed this with @rwjblue, he has a much better solution I'm going to incorporate.